test/unittests/Microsoft.ServiceFabric.Services.Wcf.Tests/WcfServiceRemotingClientTest.cs

```diff
@@ -0,0 +1,166 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Moq;
+using Xunit;
+using System;
+using System.ServiceModel;
+using System.Collections.Generic;
+using Microsoft.ServiceFabric.Services.Remoting.V2.Runtime;
+using Microsoft.ServiceFabric.Services.Communication;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Services.Remoting.V2.Client;
+using Fuzzy;
+
+namespace Microsoft.ServiceFabric.Services.Remoting.V2.Wcf.Client
+{
+    public abstract class WcfServiceRemotingClientTestBase
```

Comment:
There is usually very little value in the term Base, unless you're implying https://youtu.be/rfMC2aVhYuo :). More importantly, the .cs file name should match the name of the class, so I would drop it.